apollo-ast/api/apollo-ast.api

```diff
@@ -1115,6 +1115,7 @@ public final class com/apollographql/apollo3/ast/Schema {
 	public static final field FIELD_POLICY Ljava/lang/String;
 	public static final field FIELD_POLICY_FOR_FIELD Ljava/lang/String;
 	public static final field FIELD_POLICY_KEY_ARGS Ljava/lang/String;
+	public static final field FIELD_POLICY_PAGINATION_ARGS Ljava/lang/String;
```

Comment:
This should not be part of the public API as it's marked `@ApolloExperimental` 🤔

Reply:
I'm not sure why, but if I make it not `const` it disappears. But if I put `@JvmStatic` it comes back. Could this be a binary-compatibility-validator bug?

Reply:
Making it const removes the getter on the Companion object... I guess this is why. I'll investigate see if there's a quick fix for this.

Reply:
It's not a quick fix unfortunately :(. I opened https://github.com/Kotlin/binary-compatibility-validator/issues/90